src/compiler/build/compiler-ctx.ts

```diff
@@ -2,6 +2,7 @@ import type * as d from '../../declarations';
 import { basename, dirname, extname, join } from 'path';
 import { buildEvents } from '../events';
 import { noop, normalizePath } from '@utils';
+import { InMemoryFileSystem } from '../sys/in-memory-fs';
```

Comment:
the `InMemoryFileSystem` type is now defined in the same file as `createInMemoryFileSystem` and is also exported from `src/compiler/index.ts`. I updated imports to use relative imports _within_ the `compiler` module (i.e. in any file below `src/compiler/`) and I imported from `@stencil/core/compiler` _outside_ of the module, for the sake of consistency and, well, modularity 😄